2025-03-31T20:34:19.382566597Z + timeout 3m /bin/bash -exuo pipefail -c 'while [ -n "$(ss -Htanop \( sport = 9443 \))" ]; do sleep 1; done'
2025-03-31T20:34:19.388703019Z ++ ss -Htanop '(' sport = 9443 ')'
2025-03-31T20:34:19.394449219Z + '[' -n '' ']'
2025-03-31T20:34:19.400488935Z + exec cluster-kube-controller-manager-operator cert-recovery-controller --kubeconfig=/etc/kubernetes/static-pod-resources/configmaps/kube-controller-cert-syncer-kubeconfig/kubeconfig --namespace=openshift-kube-controller-manager --listen=0.0.0.0:9443 -v=2
2025-03-31T20:34:19.456090231Z W0331 20:34:19.456012       1 cmd.go:245] Using insecure, self-signed certificates
2025-03-31T20:34:19.456226958Z I0331 20:34:19.456213       1 crypto.go:601] Generating new CA for cert-recovery-controller-signer@1743453259 cert, and key in /tmp/serving-cert-353671612/serving-signer.crt, /tmp/serving-cert-353671612/serving-signer.key
2025-03-31T20:34:19.651583495Z I0331 20:34:19.651542       1 leaderelection.go:122] The leader election gives 4 retries and allows for 30s of clock skew. The kube-apiserver downtime tolerance is 78s. Worst non-graceful lease acquisition is 2m43s. Worst graceful lease acquisition is {26s}.
2025-03-31T20:34:19.653080976Z I0331 20:34:19.653059       1 observer_polling.go:159] Starting file observer
2025-03-31T20:34:19.653390901Z W0331 20:34:19.653372       1 builder.go:267] unable to get owner reference (falling back to namespace): Get "https://localhost:6443/api/v1/namespaces/openshift-kube-controller-manager/pods": dial tcp [::1]:6443: connect: connection refused
2025-03-31T20:34:19.653544587Z I0331 20:34:19.653535       1 builder.go:299] cert-recovery-controller version 4.15.0-202502170147.p0.gee8cf52.assembly.stream.el9-ee8cf52-ee8cf52558df0862f9927dbc7275fbe6cc1a1e5f
2025-03-31T20:34:19.654469709Z W0331 20:34:19.654455       1 builder.go:358] unable to get control plane topology, using HA cluster values for leader election: Get "https://localhost:6443/apis/config.openshift.io/v1/infrastructures/cluster": dial tcp [::1]:6443: connect: connection refused
2025-03-31T20:34:19.654666226Z I0331 20:34:19.654648       1 leaderelection.go:250] attempting to acquire leader lease openshift-kube-controller-manager/cert-recovery-controller-lock...
2025-03-31T20:34:19.655331493Z I0331 20:34:19.655310       1 event.go:298] Event(v1.ObjectReference{Kind:"Namespace", Namespace:"openshift-kube-controller-manager", Name:"openshift-kube-controller-manager", UID:"", APIVersion:"v1", ResourceVersion:"", FieldPath:""}): type: 'Warning' reason: 'ControlPlaneTopology' unable to get control plane topology, using HA cluster values for leader election: Get "https://localhost:6443/apis/config.openshift.io/v1/infrastructures/cluster": dial tcp [::1]:6443: connect: connection refused
2025-03-31T20:34:19.655526070Z E0331 20:34:19.655469       1 event.go:289] Unable to write event: '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"openshift-kube-controller-manager.1831fd80a03342fe", GenerateName:"", Namespace:"openshift-kube-controller-manager", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Namespace", Namespace:"openshift-kube-controller-manager", Name:"openshift-kube-controller-manager", UID:"", APIVersion:"v1", ResourceVersion:"", FieldPath:""}, Reason:"ControlPlaneTopology", Message:"unable to get control plane topology, using HA cluster values for leader election: Get \"https://localhost:6443/apis/config.openshift.io/v1/infrastructures/cluster\": dial tcp [::1]:6443: connect: connection refused", Source:v1.EventSource{Component:"cert-recovery-controller", Host:""}, FirstTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), LastTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), Count:1, Type:"Warning", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"cert-recovery-controller", ReportingInstance:""}': 'Post "https://localhost:6443/api/v1/namespaces/openshift-kube-controller-manager/events": dial tcp [::1]:6443: connect: connection refused'(may retry after sleeping)
2025-03-31T20:34:19.655756685Z E0331 20:34:19.655741       1 leaderelection.go:332] error retrieving resource lock openshift-kube-controller-manager/cert-recovery-controller-lock: Get "https://localhost:6443/apis/coordination.k8s.io/v1/namespaces/openshift-kube-controller-manager/leases/cert-recovery-controller-lock?timeout=1m47s": dial tcp [::1]:6443: connect: connection refused
2025-03-31T20:34:28.018969669Z E0331 20:34:28.018884       1 event.go:289] Unable to write event: '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"openshift-kube-controller-manager.1831fd80a03342fe", GenerateName:"", Namespace:"openshift-kube-controller-manager", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Namespace", Namespace:"openshift-kube-controller-manager", Name:"openshift-kube-controller-manager", UID:"", APIVersion:"v1", ResourceVersion:"", FieldPath:""}, Reason:"ControlPlaneTopology", Message:"unable to get control plane topology, using HA cluster values for leader election: Get \"https://localhost:6443/apis/config.openshift.io/v1/infrastructures/cluster\": dial tcp [::1]:6443: connect: connection refused", Source:v1.EventSource{Component:"cert-recovery-controller", Host:""}, FirstTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), LastTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), Count:1, Type:"Warning", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"cert-recovery-controller", ReportingInstance:""}': 'Post "https://localhost:6443/api/v1/namespaces/openshift-kube-controller-manager/events": dial tcp [::1]:6443: connect: connection refused'(may retry after sleeping)
2025-03-31T20:34:38.020603317Z E0331 20:34:38.020481       1 event.go:289] Unable to write event: '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"openshift-kube-controller-manager.1831fd80a03342fe", GenerateName:"", Namespace:"openshift-kube-controller-manager", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Namespace", Namespace:"openshift-kube-controller-manager", Name:"openshift-kube-controller-manager", UID:"", APIVersion:"v1", ResourceVersion:"", FieldPath:""}, Reason:"ControlPlaneTopology", Message:"unable to get control plane topology, using HA cluster values for leader election: Get \"https://localhost:6443/apis/config.openshift.io/v1/infrastructures/cluster\": dial tcp [::1]:6443: connect: connection refused", Source:v1.EventSource{Component:"cert-recovery-controller", Host:""}, FirstTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), LastTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), Count:1, Type:"Warning", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"cert-recovery-controller", ReportingInstance:""}': 'Post "https://localhost:6443/api/v1/namespaces/openshift-kube-controller-manager/events": dial tcp [::1]:6443: connect: connection refused'(may retry after sleeping)
2025-03-31T20:34:48.024509113Z E0331 20:34:48.024316       1 event.go:289] Unable to write event: '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"openshift-kube-controller-manager.1831fd80a03342fe", GenerateName:"", Namespace:"openshift-kube-controller-manager", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Namespace", Namespace:"openshift-kube-controller-manager", Name:"openshift-kube-controller-manager", UID:"", APIVersion:"v1", ResourceVersion:"", FieldPath:""}, Reason:"ControlPlaneTopology", Message:"unable to get control plane topology, using HA cluster values for leader election: Get \"https://localhost:6443/apis/config.openshift.io/v1/infrastructures/cluster\": dial tcp [::1]:6443: connect: connection refused", Source:v1.EventSource{Component:"cert-recovery-controller", Host:""}, FirstTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), LastTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), Count:1, Type:"Warning", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"cert-recovery-controller", ReportingInstance:""}': 'Post "https://localhost:6443/api/v1/namespaces/openshift-kube-controller-manager/events": dial tcp [::1]:6443: connect: connection refused'(may retry after sleeping)
2025-03-31T20:34:51.588861637Z E0331 20:34:51.588782       1 leaderelection.go:332] error retrieving resource lock openshift-kube-controller-manager/cert-recovery-controller-lock: Get "https://localhost:6443/apis/coordination.k8s.io/v1/namespaces/openshift-kube-controller-manager/leases/cert-recovery-controller-lock?timeout=1m47s": dial tcp [::1]:6443: connect: connection refused
2025-03-31T20:34:58.027466247Z E0331 20:34:58.027339       1 event.go:289] Unable to write event: '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"openshift-kube-controller-manager.1831fd80a03342fe", GenerateName:"", Namespace:"openshift-kube-controller-manager", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Namespace", Namespace:"openshift-kube-controller-manager", Name:"openshift-kube-controller-manager", UID:"", APIVersion:"v1", ResourceVersion:"", FieldPath:""}, Reason:"ControlPlaneTopology", Message:"unable to get control plane topology, using HA cluster values for leader election: Get \"https://localhost:6443/apis/config.openshift.io/v1/infrastructures/cluster\": dial tcp [::1]:6443: connect: connection refused", Source:v1.EventSource{Component:"cert-recovery-controller", Host:""}, FirstTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), LastTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), Count:1, Type:"Warning", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"cert-recovery-controller", ReportingInstance:""}': 'Post "https://localhost:6443/api/v1/namespaces/openshift-kube-controller-manager/events": dial tcp [::1]:6443: connect: connection refused'(may retry after sleeping)
2025-03-31T20:35:08.031512970Z E0331 20:35:08.031363       1 event.go:289] Unable to write event: '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"openshift-kube-controller-manager.1831fd80a03342fe", GenerateName:"", Namespace:"openshift-kube-controller-manager", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Namespace", Namespace:"openshift-kube-controller-manager", Name:"openshift-kube-controller-manager", UID:"", APIVersion:"v1", ResourceVersion:"", FieldPath:""}, Reason:"ControlPlaneTopology", Message:"unable to get control plane topology, using HA cluster values for leader election: Get \"https://localhost:6443/apis/config.openshift.io/v1/infrastructures/cluster\": dial tcp [::1]:6443: connect: connection refused", Source:v1.EventSource{Component:"cert-recovery-controller", Host:""}, FirstTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), LastTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), Count:1, Type:"Warning", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"cert-recovery-controller", ReportingInstance:""}': 'Post "https://localhost:6443/api/v1/namespaces/openshift-kube-controller-manager/events": dial tcp [::1]:6443: connect: connection refused'(may retry after sleeping)
2025-03-31T20:35:18.034108991Z E0331 20:35:18.033910       1 event.go:289] Unable to write event: '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"openshift-kube-controller-manager.1831fd80a03342fe", GenerateName:"", Namespace:"openshift-kube-controller-manager", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Namespace", Namespace:"openshift-kube-controller-manager", Name:"openshift-kube-controller-manager", UID:"", APIVersion:"v1", ResourceVersion:"", FieldPath:""}, Reason:"ControlPlaneTopology", Message:"unable to get control plane topology, using HA cluster values for leader election: Get \"https://localhost:6443/apis/config.openshift.io/v1/infrastructures/cluster\": dial tcp [::1]:6443: connect: connection refused", Source:v1.EventSource{Component:"cert-recovery-controller", Host:""}, FirstTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), LastTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), Count:1, Type:"Warning", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"cert-recovery-controller", ReportingInstance:""}': 'Post "https://localhost:6443/api/v1/namespaces/openshift-kube-controller-manager/events": dial tcp [::1]:6443: connect: connection refused'(may retry after sleeping)
2025-03-31T20:35:28.036313679Z E0331 20:35:28.036146       1 event.go:289] Unable to write event: '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"openshift-kube-controller-manager.1831fd80a03342fe", GenerateName:"", Namespace:"openshift-kube-controller-manager", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Namespace", Namespace:"openshift-kube-controller-manager", Name:"openshift-kube-controller-manager", UID:"", APIVersion:"v1", ResourceVersion:"", FieldPath:""}, Reason:"ControlPlaneTopology", Message:"unable to get control plane topology, using HA cluster values for leader election: Get \"https://localhost:6443/apis/config.openshift.io/v1/infrastructures/cluster\": dial tcp [::1]:6443: connect: connection refused", Source:v1.EventSource{Component:"cert-recovery-controller", Host:""}, FirstTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), LastTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), Count:1, Type:"Warning", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"cert-recovery-controller", ReportingInstance:""}': 'Post "https://localhost:6443/api/v1/namespaces/openshift-kube-controller-manager/events": dial tcp [::1]:6443: connect: connection refused'(may retry after sleeping)
2025-03-31T20:35:38.041464740Z E0331 20:35:38.041339       1 event.go:289] Unable to write event: '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"openshift-kube-controller-manager.1831fd80a03342fe", GenerateName:"", Namespace:"openshift-kube-controller-manager", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Namespace", Namespace:"openshift-kube-controller-manager", Name:"openshift-kube-controller-manager", UID:"", APIVersion:"v1", ResourceVersion:"", FieldPath:""}, Reason:"ControlPlaneTopology", Message:"unable to get control plane topology, using HA cluster values for leader election: Get \"https://localhost:6443/apis/config.openshift.io/v1/infrastructures/cluster\": dial tcp [::1]:6443: connect: connection refused", Source:v1.EventSource{Component:"cert-recovery-controller", Host:""}, FirstTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), LastTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), Count:1, Type:"Warning", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"cert-recovery-controller", ReportingInstance:""}': 'Post "https://localhost:6443/api/v1/namespaces/openshift-kube-controller-manager/events": dial tcp [::1]:6443: connect: connection refused'(may retry after sleeping)
2025-03-31T20:35:48.013308319Z E0331 20:35:48.010636       1 leaderelection.go:332] error retrieving resource lock openshift-kube-controller-manager/cert-recovery-controller-lock: Get "https://localhost:6443/apis/coordination.k8s.io/v1/namespaces/openshift-kube-controller-manager/leases/cert-recovery-controller-lock?timeout=1m47s": dial tcp [::1]:6443: connect: connection refused
2025-03-31T20:35:48.042580218Z E0331 20:35:48.042480       1 event.go:289] Unable to write event: '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"openshift-kube-controller-manager.1831fd80a03342fe", GenerateName:"", Namespace:"openshift-kube-controller-manager", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Namespace", Namespace:"openshift-kube-controller-manager", Name:"openshift-kube-controller-manager", UID:"", APIVersion:"v1", ResourceVersion:"", FieldPath:""}, Reason:"ControlPlaneTopology", Message:"unable to get control plane topology, using HA cluster values for leader election: Get \"https://localhost:6443/apis/config.openshift.io/v1/infrastructures/cluster\": dial tcp [::1]:6443: connect: connection refused", Source:v1.EventSource{Component:"cert-recovery-controller", Host:""}, FirstTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), LastTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), Count:1, Type:"Warning", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"cert-recovery-controller", ReportingInstance:""}': 'Post "https://localhost:6443/api/v1/namespaces/openshift-kube-controller-manager/events": dial tcp [::1]:6443: connect: connection refused'(may retry after sleeping)
2025-03-31T20:35:58.046166680Z E0331 20:35:58.045776       1 event.go:289] Unable to write event: '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"openshift-kube-controller-manager.1831fd80a03342fe", GenerateName:"", Namespace:"openshift-kube-controller-manager", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Namespace", Namespace:"openshift-kube-controller-manager", Name:"openshift-kube-controller-manager", UID:"", APIVersion:"v1", ResourceVersion:"", FieldPath:""}, Reason:"ControlPlaneTopology", Message:"unable to get control plane topology, using HA cluster values for leader election: Get \"https://localhost:6443/apis/config.openshift.io/v1/infrastructures/cluster\": dial tcp [::1]:6443: connect: connection refused", Source:v1.EventSource{Component:"cert-recovery-controller", Host:""}, FirstTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), LastTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), Count:1, Type:"Warning", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"cert-recovery-controller", ReportingInstance:""}': 'Post "https://localhost:6443/api/v1/namespaces/openshift-kube-controller-manager/events": dial tcp [::1]:6443: connect: connection refused'(may retry after sleeping)
2025-03-31T20:36:08.048060903Z E0331 20:36:08.047969       1 event.go:289] Unable to write event: '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"openshift-kube-controller-manager.1831fd80a03342fe", GenerateName:"", Namespace:"openshift-kube-controller-manager", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Namespace", Namespace:"openshift-kube-controller-manager", Name:"openshift-kube-controller-manager", UID:"", APIVersion:"v1", ResourceVersion:"", FieldPath:""}, Reason:"ControlPlaneTopology", Message:"unable to get control plane topology, using HA cluster values for leader election: Get \"https://localhost:6443/apis/config.openshift.io/v1/infrastructures/cluster\": dial tcp [::1]:6443: connect: connection refused", Source:v1.EventSource{Component:"cert-recovery-controller", Host:""}, FirstTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), LastTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), Count:1, Type:"Warning", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"cert-recovery-controller", ReportingInstance:""}': 'Post "https://localhost:6443/api/v1/namespaces/openshift-kube-controller-manager/events": dial tcp [::1]:6443: connect: connection refused'(may retry after sleeping)
2025-03-31T20:36:08.048060903Z E0331 20:36:08.048028       1 event.go:228] Unable to write event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"openshift-kube-controller-manager.1831fd80a03342fe", GenerateName:"", Namespace:"openshift-kube-controller-manager", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Namespace", Namespace:"openshift-kube-controller-manager", Name:"openshift-kube-controller-manager", UID:"", APIVersion:"v1", ResourceVersion:"", FieldPath:""}, Reason:"ControlPlaneTopology", Message:"unable to get control plane topology, using HA cluster values for leader election: Get \"https://localhost:6443/apis/config.openshift.io/v1/infrastructures/cluster\": dial tcp [::1]:6443: connect: connection refused", Source:v1.EventSource{Component:"cert-recovery-controller", Host:""}, FirstTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), LastTimestamp:time.Date(2025, time.March, 31, 20, 34, 19, 654447870, time.Local), Count:1, Type:"Warning", EventTime:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"cert-recovery-controller", ReportingInstance:""}' (retry limit exceeded!)
2025-03-31T20:36:25.454520454Z E0331 20:36:25.454454       1 leaderelection.go:332] error retrieving resource lock openshift-kube-controller-manager/cert-recovery-controller-lock: Get "https://localhost:6443/apis/coordination.k8s.io/v1/namespaces/openshift-kube-controller-manager/leases/cert-recovery-controller-lock?timeout=1m47s": dial tcp [::1]:6443: connect: connection refused
2025-03-31T20:36:51.975723073Z E0331 20:36:51.975671       1 leaderelection.go:332] error retrieving resource lock openshift-kube-controller-manager/cert-recovery-controller-lock: Get "https://localhost:6443/apis/coordination.k8s.io/v1/namespaces/openshift-kube-controller-manager/leases/cert-recovery-controller-lock?timeout=1m47s": dial tcp [::1]:6443: connect: connection refused
2025-04-01T08:16:04.499795432Z E0401 08:16:04.499710       1 leaderelection.go:332] error retrieving resource lock openshift-kube-controller-manager/cert-recovery-controller-lock: Get "https://localhost:6443/apis/coordination.k8s.io/v1/namespaces/openshift-kube-controller-manager/leases/cert-recovery-controller-lock?timeout=1m47s": dial tcp [::1]:6443: connect: connection refused
